### ISSUE TITLE
7.4 redactorx image positioning.

### ISSUE THREAD

**Issue by robinsowell** [ExpressionEngine/ExpressionEngine#4017]
This may be user error on my part, but I don't think so.

When you upload an image and add it to your text area, it's inside a 'positioning box' for lack of a better term.  It spans the full text box and you can right/left/center the image in the box, which is awesome.  But if you resize the image, it resizes the positioning box so it no longer stretches the full width, and there's no way center it relative to the surrounding text- just to the box, which is now left justified.  

Basically?  If you resize the image via redactor, it's hard/impossible to center it because the box it's in no longer is centered.

https://github.com/ExpressionEngine/ExpressionEngine/assets/1181219/40a0455f-8101-40f1-8126-8fde5fb7efb9

**Comment by intoeetive** [comment]
Confirming I can see the issue.

The changelog for RedactorX says  that "_The "image resize" plugin doesn't work correctly with the "image positioning" plugin._" issue has been fixed. 

I'll make this known to RedactorX developers
